docs/book/src/reference/completion.md

```diff
@@ -0,0 +1,53 @@
+# Enabling shell auto completion
+
+## Bash on MacOS
+The kubebuilder completion script for Bash can be generated with kubebuilder completion bash. Sourcing this script in your shell enables kubebuilder completion.
+
+However, the kubebuilder completion script depends on bash-completion v2 and Bash 4.1+, which you thus have to previously install.
+
+### Upgrade Bash
+#### Upgrade to  Bash 4.1+. You can check your Bash’s version by running:
+
+`echo BASH_VERSION`
+
+#### To upgrade, you may use homebrew:
```

Comment:
If these are the steps then, I do not think that we should use ### since it is for section titles.

Reply:
See an example of the layout applied in another place: https://deploy-preview-1437--kubebuilder.netlify.com/reference/kind.html#cheetsheet